integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/carbonserver/CarbonServerManager.java

```diff
@@ -223,9 +223,13 @@ private void copyResources() throws IOException {
         String srcDirectory =
                 (System.getProperty("basedir", ".")) + File.separator + "target" + File.separator + "samples"
                         + File.separator;
-        log.info("Copying resources from " + srcDirectory);
-        FileUtils.copyDirectoryToDirectory(new File(srcDirectory), new File(carbonHome));
-        log.info("Completed copying resources");
+        File srcFile = new File(srcDirectory);
+
+        if ( srcFile.exists() ) {
```

Comment:
Can the src file not exist?

Reply:
Yes, We don't need the _samples_ resources for this cli-module. Hense needs to add this check before copying the resources.